tests/management/role/test_dual_write.py

```diff
@@ -444,6 +444,49 @@ def test_adding_same_role_again_and_unassign_it_once(self):
         )
         self.assertEquals(len(tuples), 0)
 
+    def test_group_not_bound_after_reset_when_role_added_multiple_times(self):
```

Comment:
>  From jira Given we add the same role to group dual write handler multiple times, and then reset the role, we should no longer see group bound, and the tuple should be returned for that group relation

Is correct that we should not see group bound ? 
when reset group-roles assignments and there is group-roles assignment, should be group still in mapping ?

Reply:
Yes, it should be. Reset makes sure it only gets added once.